Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -382,6 +377,8 @@ void BarOnNavigationClick(object sender, AToolbar.NavigationClickEventArgs navig
 
 		void CurrentOnPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
+			if(e.PropertyName == NavigationPage.InternalPaddingProperty.PropertyName)
+				Element.ForceLayout();
```

Comment:
@StephaneDelcroix There is not RequestLayout in NavigationPage, only ForceLayout